app/views/components/widget/_fba.js.erb

```diff
@@ -82,12 +86,28 @@ function FBAform(d, N) {
 			});
 
 		},
+		addModalJs() {
+			const scriptElement = document.createElement('script');
+			scriptElement.type = 'text/javascript';
+			scriptElement.defer = true;
+			scriptElement.src = "<%= asset_url('touchpoints-uswds.min.js') %>"
```

Comment:
one consideration here is to see if this asset gets hashed differently each deploy.

if so, I'm wondering about a scenario where a client copies this code to host on their own, but the asset reference goes stale between deployments.